test/check-application

```diff
@@ -1948,14 +1948,14 @@ class TestFiles(testlib.MachineCase):
         b.wait_visible("[data-item='testdir']")
 
         # Manually edit path
-        b.key("L", modifiers=["Control"])
+        b.key("L", modifiers=["Control", "Shift"])
```

Comment:
:thinking: the keybindings didn't change afaik?

Reply:
Is this some regression in uhhh bidi?

Reply:
Keybind didnt change and it's not any sorts of bidi regression. The proper keybind is `ctrl` + `shift` + `L` so lets make bidi do it 100% right. 
Previously it was fine because the check for keyboard modifiers didn't look at shift modifier and only relied on uppercase L. Now the check is more robust.

Reply:
Ok!